openmessaging-transport-binding.md

```diff
@@ -0,0 +1,289 @@
+# OpenMessaging Transport Binding for CloudEvents
+
+## Abstract
+
+The [OpenMessaging][OpenMessaging] Transport Binding for CloudEvents defines how
+events are mapped to [OpenMessaging Specification][OpenMessaging-Spec].
+
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+- 1.1. [Conformance](#11-conformance)
+- 1.2. [Relation to OpenMessaging](#12-relation-to-openmessaging)
+- 1.3. [Content Modes](#13-content-modes)
+- 1.4. [Event Formats](#14-event-formats)
+- 1.5. [Security](#15-security)
+2. [Use of CloudEvents Attributes](#2-use-of-cloudevents-attributes)
+- 2.1. [contentType Attribute](#21-contenttype-attribute)
+- 2.2. [data Attribute](#22-data-attribute)
+3. [OpenMessaging Message Mapping](#3-openmessaging-message-mapping)
+- 3.2. [Binary Content Mode](#31-binary-content-mode)
+- 3.1. [Structured Content Mode](#32-structured-content-mode)
+4. [References](#4-references)
+
+## 1. Introduction
+
+This specification defines how
+the elements defined in the CloudEvents specification are to be used in
+[OpenMessaging][OpenMessaging]. 
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][RFC2119].
+
+### 1.2. Relation to OpenMessaging
+
+This specification does not prescribe rules constraining transfer or settlement
+of event messages with [OpenMessaging][OpenMessaging]; it solely defines how 
+CloudEvents are expressed with [OpenMessaging 
+Specification][OpenMessaging-Spec].    
+
+OpenMessaging-based messaging and eventing infrastructures may provide
```

Comment:
its not clear if the "may" and "optional" are meant to be RFC2119 keywords here or not. If they are, can you upper-case them? You should add these md files to the checker in the Makefile too.